<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-LNTPU46U" failures="1" tests="1" name="Test" time="4.669" errors="0" timestamp="2024-10-17T11:53:53 IST">
  <testcase classname="testCases.TC003_LoginDDT" name="verify_LoginDDT" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.lang.NullPointerException: Cannot invoke &amp;quot;java.io.FileOutputStream.close()&amp;quot; because &amp;quot;this.fo&amp;quot; is null">
      <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "java.io.FileOutputStream.close()" because "this.fo" is null
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.NullPointerException: Cannot invoke "java.io.FileOutputStream.close()" because "this.fo" is null
at utilities.ExcelUtility.getCellCount(ExcelUtility.java:53)
at utilities.DataProviders.getData(DataProviders.java:21)
... 25 more
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verify_LoginDDT -->
</testsuite> <!-- Test -->
